lib/charms/grafana_k8s/v0/grafana_source.py

```diff
@@ -171,6 +172,22 @@ def __init__(self, *args):
 RELATION_INTERFACE_NAME = "grafana_datasource"
 
 
+@dataclass
+class GrafanaSourceData:
+    """This class represents the data Grafana provides others about itself."""
+
+    datasource_uids: Dict[str, str]
+    external_url: str
+
+    def get_unit_uid(self, unit: str):
+        """Return the UID for a given unit."""
+        if unit in self.datasource_uids:
+            datasource_uid = self.datasource_uids[unit]
+        else:
+            datasource_uid = ""
```

Comment:
Just a minor style change in this Grafana lib:

```suggestion
        datasource_uid = ""
        if unit in self.datasource_uids:
            datasource_uid = self.datasource_uids[unit]            
```

Reply:
Unfortunately, already merged the PR on Grafana lib. Do you think this is worth a new lib version bump?